apiary.apib

```diff
@@ -3518,6 +3518,60 @@ Update the current timer. Only possible if there is a timer running.
 
 ## Work Orders [/workOrders]
 
+### workOrders.info [GET /workOrders.info]
+
+Get details for a single work order.
+
++ Request (application/json;charset=utf-8)
+
+    + Attributes (object)
+        + id: `cde0bc5f-8602-4e12-b5d3-f03436b54c0d` (string, required)
+
++ Response 200 (application/json;charset=utf-8)
+
+    + Attributes (object)
+        + data (object)
+            + id: `cde0bc5f-8602-4e12-b5d3-f03436b54c0d` (string)
+            + state: `draft` (enum[string])
+                + Members
+                    + draft
+                    + finalized
+                    + sent
+            + customer (object)
+                + type (enum[string])
+                    + Members
+                        + contact
+                        + company
+                + id: `09e5d75f-f817-4872-9723-57fbb8ff710d` (string)
+            + starts_at: `2017-05-26T10:01:49+00:00` (string)
```

Comment:
Q: This feels a bit weird, what if the work order was already completed?

Reply:
Should be in the same tense as the `started_at` of the work_hours imo

Reply:
We already name this parameter like this on `workOrders.draft`. 
If we change it here, to be consistent, we should name it there also. 
But altering it now would be a BC.

Reply:
We discussed this during (long) grooming sessions and this is the most non-restrictive term. Using a past tense doesn't make sense when you draft work orders to be executed days later. Work hours _may_ be drafted as well, but there we took the format from the time tracking endpoints.